### PR TITLE
svg_loader SvgLoader: Print inefficient elements

### PR DESCRIPTION
**Pull request by JSUYA**
- Description :
Opacity is 0
both Fill.Opacity and Stroke Opacity are 0
point is 0 in Path
declared display="none"
width or height of bounds becomes 0

- Issue Tickets (if any) :
https://github.com/Samsung/thorvg/issues/153

### CODE REVIEW

File: src/loaders/svg/tvgSvgLoaderCommon.h [hunk 1]
```diff
@@ -347,4 +347,33 @@ struct SvgLoaderData
     bool result = false;
 };
 
+#ifdef THORVG_LOG_ENABLED
+inline string nodeTypeToString(SvgNodeType type)
```
Comment (hermet) [inline src/loaders/svg/tvgSvgLoaderCommon.h:351]: does it necessary to use inline? leave to compiler if possible
Also, if it uses only in _inefficientNodeCheck(), you can move it in one same place.

#ifdef THORVG_LOG_ENABLED
static string _nodeTypeToString()
{
...
}
static void _inefficientNodeCheck()
{
...
}
#endif

File: src/loaders/svg/tvgSvgLoader.cpp [hunk 2]
```diff
@@ -2225,9 +2225,49 @@ static void _styleInherit(SvgStyleProperty* child, SvgStyleProperty* parent)
 }
 
 
+#ifdef THORVG_LOG_ENABLED
+void _inefficientNodeCheck(SvgNode* node){
```
Comment (hermet) [inline src/loaders/svg/tvgSvgLoader.cpp:2229]: static void

**Review by hermet — CHANGES_REQUESTED** [review]
Please check comments

File: src/loaders/svg/tvgSvgLoaderCommon.h (hunk 1 above)
Comment (JSUYA) [reply src/loaders/svg/tvgSvgLoaderCommon.h]: Removed inline
and _nodeTypeToString is also used in XmlParser.cpp. Therefore, it cannot be moved to tvgSvgLoade.

File: src/loaders/svg/tvgSvgLoader.cpp (hunk 2 above)
Comment (JSUYA) [reply src/loaders/svg/tvgSvgLoader.cpp]: Changed

**Review by hermet — APPROVED** [review]

**Pull request merged**